docs/guides/wifi.md

```diff
@@ -1,120 +1,52 @@
-    - On Linux: `cat /sys/devices/virtual/dmi/id/product_name`
-- If unknown, your Mac's wifi chipset number can be found like this:
-    - On macOS: `ioreg -l | grep RequestedFiles`, the number will be
-      within the folder names, i.e. "C-`4364`\_\_s-B3" (Your model's
-      island will also be in this command's output).
-    - On Linux: `lspci -d '14e4:*'`, this will also tell you the
-      "Rev"/Revision.
+2. Open the terminal and run `cp -r /usr/share/firmware/wifi ~/Desktop`. This shall make a folder named **wifi** on the macOS Desktop.
 
-| Model Identifier | Chipset | Revision | Island   | Firmware Options     |
-|------------------|---------|----------|----------|----------------------|
-| MacBookPro16,1   | BCM4364 | 4        | Bali     | Big Sur              |
-| MacBookPro16,2   | BCM4364 | 4        | Trinidad | Big Sur              |
-| MacBookPro16,3   | BCM4377 | 4        | Tahiti   | Big Sur              |
-| MacBookPro16,4   | BCM4364 | 4        | Borneo   | Big Sur              |
-| MacBookPro15,1   | BCM4364 | 3        | Kauai    | Mojave / Big Sur     |
-| MacBookPro15,2   | BCM4364 | 3        | Maui     | Mojave / Big Sur     |
-| MacBookPro15,3   | BCM4364 | 3        | Kauai    | Mojave / Big Sur     |
-| MacBookPro15,4   | BCM4377 | 4        | Formosa  | Big Sur              |
-| MacBookAir9,1    | BCM4377 | 4        | Fiji     | Big Sur              |
-| MacBookAir8,1    | BCM4355 | 12       | Hawaii   | Mojave / Big Sur     |
-| MacBookAir8,2    | BCM4355 | 12       | Hawaii   | Mojave / Big Sur     |
-| MacMini8,1       | BCM4364 | 3        | Lanai    | Mojave / Big Sur     |
-| MacPro7,1        | BCM4364 | 4        | Kahana   | Big Sur              |
-| iMac20,1         | BCM4364 | 4        | Hanauma  | Big Sur              |
-| iMac20,2         | BCM4364 | 4        | Kure     | Big Sur              |
-| iMac19,1*        | BCM4364 | 3        | Midway   | Mojave / Big Sur     |
-| iMac19,2*        | BCM4364 | 3        | Nihau    | Mojave / Big Sur     |
-| iMacPro1,1       | BCM4364 | ?        | Ekans?   | Mojave / Big Sur?    |
+3. Copy the **wifi** folder to an external drive or a partition which is formatted to a file system readable by Linux.
 
-\*iMac19,x don't have the T2 chip, but they have the same type of wifi chip as T2 Macs.
+4. Boot into Linux.
 
-*If there is missing/uncertain information for your model, please open a [pull request](https://github.com/t2linux/wiki/edit/master/docs/guides/wifi.md) or [issue](https://github.com/t2linux/wiki/issues/new) or mention it on the [discord](https://discord.com/invite/68MRhQu).*
+5. Copy the **wifi** folder to the **Home** folder using your file manager.
 
-### Retrieving Firmware
+6. Open the terminal and run :-
 
-Run `ioreg -l | grep RequestedFiles` in macOS Terminal. Note down the output, and make sure you also know your Model Identifier. The output will be similar to this:
+   ```sh
+   git clone https://github.com/AsahiLinux/asahi-installer --depth=1
```

Comment:
I'd prefer to have this bit done on macOS, as that's the process that Marcan had in his [summary patch](https://lore.kernel.org/all/20211226153624.162281-1-marcan@marcan.st/):

> To test this series on a supported Mac today (T2 or M1), boot into macOS
> and run:
>
> $ git clone https://github.com/AsahiLinux/asahi-installer
> $ cd asahi-installer/src
> $ python -m firmware.wifi /usr/share/firmware/wifi firmware.tar
>
> Then copy firmware.tar to Linux and extract it into /lib/firmware.

I think once asahi-installer gets a graphical interface, it'll be doing it from macOS automatically as marcan described, so that could break more of the procedure that we currently have.

Reply:
Consider the fact that not all users use Xcode and Homebrew

Reply:
Also, the sole purpose of writing this guide is to get things working as per the current situation. We shall keep it more updated when the situation gets better from Asahi’s side. Probably, the Asahi installer won’t demand git and python for Mac, or they’ll put some documentation to use it which I am not aware about.

Reply:
I forgot that git isn't installed by default (and i supposed python too). That makes sense then.